### DISCUSSION TITLE
Quick Picks Error

### DISCUSSION THREAD

**Discussion by omarblack21** [fast4x/RiMusic#3521] [General]
After I Updated RiMusic .in quick picks or for you .Nothing showed up for me except the last song I played recently. ![Screenshot_2024-09-02-17-51-21-075_it.fast4x.rimusic.jpg](https://github.com/user-attachments/assets/237f4c33-1304-44de-9687-aeeca8e4f9fc)

**Comment by fast4x** [comment]
Download custom version, it's Just fixed
